ci has been failing for centuries, figure out why

Installing teal seems to be the issue?

```
cyan dev-1 depends on tl (not installed)
Installing https://luarocks.org/dev/tl-dev-1.rockspec

Error: Failed installing dependency: https://luarocks.org/dev/tl-dev-1.rockspec - Failed cloning git repository.
Error: Process completed with exit code 1.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -21,12 +21,12 @@ jobs:
       - uses: actions/checkout@main
 
       - name: Install Lua
-        uses: leafo/gh-actions-lua@v8.0.0
+        uses: leafo/gh-actions-lua@v10.0.0
         with:
           luaVersion: ${{ matrix.luaVersion }}
 
       - name: Install LuaRocks
-        uses: leafo/gh-actions-luarocks@v4.0.0
+        uses: leafo/gh-actions-luarocks@v4.3.0
 
       - name: Build
         run: |

